lib/index.js

```diff
@@ -8,11 +9,10 @@ var fs = require('fs'),
  */
 
 function getBinding() {
-  var name = process.platform + '-' + process.arch;
   var candidates = [
     path.join(__dirname, '..', 'build', 'Release', 'binding.node'),
     path.join(__dirname, '..', 'build', 'Debug', 'binding.node'),
```

Comment:
@xzyfer, @kevva ,
I think Debug should take precedence over Release? because if you explicitly build Debug binary, then your primary intent is to debug that binary in debug mode. :)